modules/ismp/pallets/pallet/README.md

```diff
@@ -32,7 +32,7 @@ To use it in your runtime, you need to implement the ismp
 ### Dispatchable Functions
 
 - `handle` - Handles incoming ISMP messages.
-- `handle_unsigned` Unsigned variant for handling incoming messages, enabled by `feature = ["unsigned"]`
+- `handle_unsigned` Unsigned variant for handling incoming messages
```

Comment:
We would need to also consolidate these calls

Reply:
For instance, depending on how much backward compatibility you want to preserve, we could optionally take out index 0 and start them `handle`, but if you think that will break people, we should leave handle at 1
```
/// Execute the provided batch of ISMP messages. This call will short-circuit and revert if
		/// any of the provided messages are invalid.
		///
		/// The dispatch origin for this call must be an unsigned one.
		///
		/// - `messages`: A set of ISMP [`Message`]s to handle or process.
		///
		/// Emits different message events based on the Message received if successful.
		#[pallet::weight(get_weight::<T>(&messages))]
		#[pallet::call_index(1)]
		#[frame_support::transactional]
		pub fn handle(origin: OriginFor<T>, messages: Vec<Message>) -> DispatchResultWithPostInfo {
			if EnableUnsigned::<T>::get() {
				ensure_none(origin)?;
			} else {
				ensure_signed(origin)?;
			}

			Self::handle_messages(messages)
		}```